powershell/plan.ps1

```diff
@@ -0,0 +1,34 @@
+$pkg_name="powershell"
+$pkg_origin="core"
+$pkg_version="6.0.0-alpha.14"
+$pkg_license=@('MIT')
```

Comment:
newb powershell question: There's single quotes here and double quotes in most other places. Is there any difference in powershell? Do you have a preference? Is there anything like ShellCheck for PowerShell? 

I do lots of Ruby programming, so I can't get very far before stopping to argue about single or double quotes. :D